.golangci.yml

```diff
@@ -31,8 +31,9 @@ linters:
   disable-all: true
   enable:
     - govet
-    - errcheck
-    - structcheck
-    - golint
-    - gofmt
-    - staticcheck
+
+linters-settings:
+  govet:
+    disable:
+      - copylocks
+      - printf
```

Comment:
Once `tests` is enabled, the `tests` `go vet` check needs to be disabled until #5764 is fixed.